factory/usr/lib/systemd/system/systemd-bootchart.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Boot Process Profiler
+Documentation=man:systemd-bootchart.service(1) man:bootchart.conf(5)
+DefaultDependencies=no
+
+[Service]
+ExecStartPre=/usr/sbin/mkdir -p /run/log
+ExecStart=/lib/systemd/systemd-bootchart -r
+KillSignal=SIGHUP
+
+[Install]
```

Comment:
if this is disabled by default, it shouldn't have any Install section, should it?
Or is it somehow needed?

Reply:
Right, not really needed. I've removed the install section.